### PR TITLE
Work section and about section

### PR DESCRIPTION
**Pull request by ntimanu**
In this project we:

- Added Work section
- Added About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+            sign-ups required.
+          </p>
+          <div class="lang">
+            <ul>
+              <li class="langtxt" id="htmltxt">html</li>
+              <li class="langtxt" id="csstxt">css</li>
+              <li class="langtxt" id="javascripttxt">javaScript</li>
+            </ul>
+          </div>
+          <button class="btntxt">See Project</button>
```
Comment (Kamba56) [inline index.html:76]: - [x] Great job adding the hover and active state on your buttons. I noticed that your button's background color doesn't properly match the Figma template. You are expected to stick to design as much as possible and this includes colors and backgrounds. Kindly fix this, please.
### Your design
image
### FIgma
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @ntimanu, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using grid and flexbox to layout contents of page👏👏👏
- [x] Great job sticking to design guidelines

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ntimanu) [reply index.html:76]: Done

**Review by BregornOriginal — APPROVED** [review]
Hi @ntimanu 

Your project is complete!  There is nothing else to say other than... it's time to merge it

Congratulations! 

giphy (7)

## Highlights

✅ Linters are passing
✅ Excellent commit history 🥇 


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**